dvc/repo/experiments/executor/base.py

```diff
@@ -450,9 +473,32 @@ def _repro_args(cls, dvc):
             kwargs = {}
         return args, kwargs
 
+    @staticmethod
+    def _auto_push(
+        dvc: "Repo",
+        scm: "Git",
+        git_remote: Optional[str],
+        push_cache=True,
+        run_cache=True,
+    ):
+        branch = scm.get_ref(EXEC_BRANCH, follow=False)
+        try:
+            dvc.experiments.push(
+                git_remote,
+                branch,
+                push_cache=push_cache,
+                run_cache=run_cache,
+            )
+        except BaseException:
+            logger.warning(
+                "Something went wrong while auto pushing checkpoint cache "
+                f"to the remote {git_remote}"
+            )
```

Comment:
```suggestion
        except BaseException as exc:
            logger.warning(
                "Something went wrong while auto pushing experiment "
                f"to the remote '{git_remote}': {exc}"
            )
```

Reply:
We need to log the exception itself so that the user can see why the push failed